renovate.json

```diff
@@ -50,6 +50,11 @@
       "matchPackageNames": ["fancy-test"],
       "allowedVersions": "1.x",
       "description": "Wait for Oclif 2 TODO:CDX-746"
+    },
+    {
+      "matchPackageNames": ["npm"],
+      "allowedVersions": "8.4.x",
+      "description": "Need NPM to be fixed https://github.com/npm/cli/issues/4404 TODO:CDX-815"
     }
```

Comment:
But isn't that the PR associated to CDX-815?

Reply:
derp